src/pages/public/HelperRegister.tsx

```diff
@@ -0,0 +1,358 @@
+import { TextField } from "@material-ui/core";
+import classnames from "classnames";
+import { Formik, FormikProps } from "formik";
+import React from "react";
+import { Briefcase, CheckCircle, Info, User } from "react-feather";
+import { useHistory } from "react-router-dom";
+import { ReactComponent as EmailSentSvg } from "~/assets/icons/EmailSent.svg";
+import { PrimaryButton } from "~/components/Button";
+import { FormTitle } from "~/components/Form";
+import { Checkbox, Radio } from "~/components/Form/components";
+import { CheckIcon } from "~/components/Icon";
+import { Step, StepContent, useSteps } from "~/components/Steps";
+import { Title } from "~/components/Title";
+
+type FormProps = FormikProps<typeof initialValues>;
+type PassedFormProps = Pick<FormProps, "setFieldValue" | "values">;
+
+const checklistTexts = [
+  "Über 18 Jahre sein",
+  "In körperlich guter Verfassung sein"
+];
+
+export const steps: Step[] = [
+  {
+    title: "Hinweis",
+    Icon: Info,
+    okText: "Okay, verstanden"
+  },
+  {
+    title: "Über dich",
+    Icon: User,
+    okText: "Weiter"
+  },
+  {
+    title: "Konditionen",
+    Icon: Briefcase,
+    okText: "Weiter"
+  },
+  {
+    title: "Fertig!",
+    Icon: CheckCircle,
+    okText: "Weiter"
+  }
+];
+
+const contents: (StepContent | StepContent[])[] = [
+  {
+    stepIndex: 0,
+    Content: () => (
+      <>
+        <p className="mb-8">
+          Bevor du fortfährst: Zuverlässigkeit steht an oberster Stelle! Wenn du
+          kurzfristig keine Zeit hast, sag bitte direkt der Kontaktperson ab,
+          damit sie umplanen kann. Weiterhin musst du:
+        </p>
+        <ul className="mb-4">
+          {checklistTexts.map((text, i) => (
+            <li key={i} className="flex align-top pl-8 mb-2">
+              <div className="inline-block -ml-8 mr-2">
+                <CheckIcon />
+              </div>
+              <p className="inline-block font-medium">{text}</p>
+            </li>
+          ))}
+        </ul>
+      </>
+    )
+  },
+
+  {
+    stepIndex: 1,
+    Content: ({ setFieldValue, values }: PassedFormProps) => (
+      <>
+        {[
+          ["account.firstName", "Vorname"],
+          ["account.lastName", "Nachname"],
+          ["account.phone", "Telefonnummer"],
+          ["account.email", "E-Mail"],
+          ["account.password", "Passwort"],
+          ["account.password_confirmation", "Passwort wiederholen"]
+        ].map(([key, label]) => (
+          <TextField
+            key={key}
+            label={label}
+            className="mb-4 text-brand"
+            fullWidth
+            onChange={(e) => setFieldValue(key, e.currentTarget.value)}
+          />
```

Comment:
Die Passwörter werden als einfacher Text-Input dargestellt. Wir sollten das ändern.

```suggestion
          ["account.firstName", "Vorname"],
          ["account.lastName", "Nachname"],
          ["account.phone", "Telefonnummer"],
          ["account.email", "E-Mail", "email"],
          ["account.password", "Passwort", "password],
          ["account.password_confirmation", "Passwort wiederholen","password"]
        ].map(([key, label, type = "text"]) => (
          <TextField
            key={key}
            type={type}
            label={label}
            className="mb-4 text-brand"
            fullWidth
            onChange={(e) => setFieldValue(key, e.currentTarget.value)}
          />
```